scripts/release_scripts/generate_release_updates.py

```diff
@@ -159,7 +159,12 @@ def create_group_for_next_release():
         'the release co-ordinator & QA Lead for next release. '
         'Please follow the release process doc: '
         '[Add link to release process doc] to ensure the release '
-        'follows the schedule. Thanks!\n' % (
+        'follows the schedule. Thanks!\n\n'
+        '3. Send the following message to oppia-dev:\n\n'
+        'Hi all, We have completed the release for current month. '
+        'This is a reminder to fill in the job requests here: '
+        'https://goo.gl/forms/XIj00RJ2h5L55XzU2 if you are planning '
```

Comment:
Should we move this URL to release constant? (In future if we want to move all the release constant to a  separate repo we won't have to do audits/changes here.)

[I'm fine with leaving it here as well, as it's not that important.]

Reply:
Also, should we ask the release co-ordinator to "Allow accepting response" to the form and disable it after release cut? (Not sure whether we use a separate form for running jobs on the backup server.)

Reply:
Moved the url.

I don't think we should disable the submission - we can instead edit the form to select target release based on the date when the form is filled (like if the form is filled just before a week of April's release, we do not allow the option for the April release to be selected). What do you think?

Reply:
Sorry, I thought we had separate forms for a running job on backup-server and prod server. As we have one form for both I agree we cant disable it.

> we can instead edit the form to allow target release selection based on the date when the form is filled

yeah, I think that's fine, as in general, we expect to write a job when we want to run it.

Reply:
Yes, I have updated the [form](https://docs.google.com/forms/d/1XqPWCEHhTIP3H_Y4snQ-_z3Pcq105PkzkUrOZKWrRFc/edit) to have a message which says that requests for current release will be rejected if they are not made at least 7 days before the first Friday of the month.